k8s/deployment.yml.j2

```diff
@@ -7,7 +7,7 @@ spec:
   selector:
     matchLabels:
       run: faasm-openmpi
-  replicas: 2
+  replicas: 4
```

Comment:
Committing this change as our baseline experiment now is four machines, from 1 to 16 MPI procs.